src/Cli.Tests/EndToEndTests.cs

```diff
@@ -768,4 +768,30 @@ public void TestBaseRouteIsConfigurableForSWA(string authProvider, bool isExcept
             Assert.AreEqual("/base-route", runtimeConfig.Runtime.BaseRoute);
         }
     }
+
+    /// <summary>
+    /// Test to validate that whenever the flag rest.request-body-strict is included in the init command,
+    /// the runtimeconfig is initialized such that the runtime allows extraneous fields in the request body for REST.
+    /// </summary>
+    /// <param name="isRequestBodyStrict">Value of the rest.request-body-strict flag in the init command.</param>
```

Comment:
nit: `param` description for `includeRestRequestBodyStrictFlag` needs to be added